### PR TITLE
AVX128: Enable all the things

### PR DESCRIPTION
**Pull request by Sonicadvance1**
Here we are, at the summit of the mountain.

### CODE REVIEW

File: unittests/ASM/VEX/bextr2.asm [hunk 1]
```diff
@@ -0,0 +1,23 @@
+%ifdef CONFIG
```
Comment (alyssarosenzweig) [inline unittests/ASM/VEX/bextr2.asm:1]: Drop this file, it duplicates the FEX_bug/ test I added for this today

Comment (Sonicadvance1) [reply unittests/ASM/VEX/bextr2.asm]: Oops, yep, added some ASM tests on accident.

File: unittests/InstructionCountCI/AVX128/VEX_map1.json [hunk 2]
```diff
@@ -0,0 +1,6318 @@
+{
+  "Features": {
+    "Bitness": 64,
+    "EnabledHostFeatures": [],
+    "DisabledHostFeatures": [
+      "FCMA",
+      "RPRES",
+      "AFP",
+      "FLAGM",
+      "FLAGM2",
```
Comment (alyssarosenzweig) [inline unittests/InstructionCountCI/AVX128/VEX_map1.json:10]: Could we enable flagm for the avx128 instcountci? It should save a few instructions on vptest.

(I've considered removing non-flagm instcountci. Even if we're not going to raise our minspec for FEX doesn't mean we need to have instcountci for it... But that's a tangential discussion to be had)

File: unittests/InstructionCountCI/AVX128/VEX_map1.json [hunk 3]
```diff
@@ -0,0 +1,6318 @@
...
+        "Map 1 0b01 0x12 128-bit"
+      ],
+      "ExpectedArm64ASM": [
+        "mov w20, #0xde4",
+        "movk w20, #0x1, lsl #16",
+        "str x20, [x28, #272]",
+        "mov w1, #0x401",
+        "str x1, [x28, #1328]",
+        "ldr x0, [x28, #2296]",
+        "br x0"
```
Comment (alyssarosenzweig) [inline unittests/InstructionCountCI/AVX128/VEX_map1.json:353]: I think something is very wrong here.

Comment (alyssarosenzweig) [reply unittests/InstructionCountCI/AVX128/VEX_map1.json:353]: I think the instcountci commit needs to be interchanged with the "enable AVX" commit.

File: unittests/InstructionCountCI/AVX128/VEX_map1.json (hunk 2 above)
Comment (Sonicadvance1) [reply unittests/InstructionCountCI/AVX128/VEX_map1.json:10]: Added the same flagm map2 file that we had for SVE256, but this time for asimd

File: unittests/InstructionCountCI/AVX128/VEX_map1.json (hunk 3 above)
Comment (Sonicadvance1) [reply unittests/InstructionCountCI/AVX128/VEX_map1.json]: Oops, instcountci was disabling AVX if SVE256 was explicitly disabled. Weird interaction that no longer needs to exist.

File: unittests/InstructionCountCI/AVX128/VEX_map1.json (hunk 2 above)
Comment (alyssarosenzweig) [reply unittests/InstructionCountCI/AVX128/VEX_map1.json:10]: v(u)comiss notably missing

Comment (Sonicadvance1) [reply unittests/InstructionCountCI/AVX128/VEX_map1.json:10]: You're right, that was in the other map file. Added that one as well.

**Comment by Sonicadvance1** [comment]
Alright CI is green. OpenSSL is still angry with AVX overall but I hope that I can find the bug in their bignum library sooner rather than later.

**Review by alyssarosenzweig — APPROVED** [review]
Let's gooooooooo

File: unittests/InstructionCountCI/AVX128/VEX_map1.json (hunk 2 above)
Comment (alyssarosenzweig) [reply unittests/InstructionCountCI/AVX128/VEX_map1.json:10]: Looks good, thx

**Pull request merged**